packages/wallets/src/components/DatePicker/__tests__/DatePicker.spec.tsx

```diff
@@ -98,10 +97,13 @@ describe('DatePicker Component', () => {
         const calendarButton = screen.getByTestId('wallets_datepicker_button');
         fireEvent.click(calendarButton);
 
-        const testDay = moment().format('D');
-        const dateElements = screen.getAllByText(testDay);
+        const date = new Date();
+        const month = `0${date.getMonth() + 1}`.slice(-2);
```

Comment:
Wait what happens if its October, November, December? Will it be `010` `011` `012`